internal/local/command_kafka_broker_describe.go

```diff
@@ -0,0 +1,42 @@
+package local
+
+import (
+	"context"
+
+	"github.com/spf13/cobra"
+
+	"github.com/confluentinc/cli/v3/pkg/broker"
+	pcmd "github.com/confluentinc/cli/v3/pkg/cmd"
+	"github.com/confluentinc/cli/v3/pkg/errors"
+	"github.com/confluentinc/cli/v3/pkg/examples"
+)
+
+func (c *command) newKafkaBrokerDescribeCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "describe [id]",
+		Short: "Describe a local Kafka broker.",
+		Long:  "Describe per-broker configuration values.",
+		Args:  cobra.MaximumNArgs(1),
```

Comment:
Should be required now that `--all` is removed.
```suggestion
		Use:   "describe <id>",
		Short: "Describe a local Kafka broker.",
		Long:  "Describe per-broker configuration values.",
		Args:  cobra.ExactArgs(1),
```